packages/react-components/src/components/table-output-component.tsx

```diff
@@ -609,6 +609,10 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
                 if (currRowIndexFound && !isFound && rowNode.data && rowNode.data['isMatched']) {
                     this.gridApi?.ensureIndexVisible(rowNode.rowIndex);
                     this.selectStartIndex = this.selectEndIndex = rowNode.rowIndex;
+                    if (this.timestampCol) {
+                        this.startTimestamp = BigInt(rowNode.data[this.timestampCol]);
+                        this.endTimestamp = BigInt(rowNode.data[this.timestampCol]);
+                    }
```

Comment:
It only works if the match is found in the current cache. Otherwise it find the index below and ensures it's made visible. We probably need for the lines to be loaded before we can update the selected time and fire the selection changed signal.

Reply:
Thanks for finding this.